### ISSUE TITLE
5.2.6 hides everything from reponse for a domain

### ISSUE THREAD

**Issue by ppisar** [rfc1036/whois#27]
whois-6.2.6 broke hiding disclaimers. ''whois -H at'' does not show anything. Reverting commit

```
commit 78d2e38a6416b5e2889eac6868afd7c05c384b57
Author: Marco d'Itri <md@linux.it>
Date:   Mon Mar 23 04:23:29 2015 +0100

    Update the disclaimer detection strings
    
    When possible they have been future-proofed by hiding everything until
    the end of the input.
```
helps.

I think it's because many of the odd hide_strings[] members changed from empty string to NULL which means to hide everything to the end of the response.

**Comment by ppisar** [comment]
This patch fixes it:

```
From b41ad469f975f274c2f2e0716241467ff495ff2b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Tue, 24 Mar 2015 17:13:05 +0100
Subject: [PATCH] Remove bogus hide_strings[] entry
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The empty starting string matched always and the final NULL string
enabled hiding to the end of the response.

As a result any response got hidden.

<https://github.com/rfc1036/whois/issues/27>

Signed-off-by: Petr Písař <ppisar@redhat.com>

diff --git a/data.h b/data.h
index 402fce5..e5c7b01 100644
--- a/data.h
+++ b/data.h
@@ -72,7 +72,6 @@ const char *hide_strings[] = {
     "TERMS OF USE: The information in the Whois database", NULL, /* .wang */
     "The WHOIS service offered by Neustar, Inc, on behalf", NULL,
     "The WHOIS service offered by the Registry Operator", NULL, /* .science */
-    "", NULL,
 
     /* ccTLDs */
     "Access to CCTLD WHOIS information is provided", "",	/* Afilias */
-- 
2.1.0

```

**Comment by rfc1036** [comment]
Fixed in 5.2.7.
